src/consensus/validation.h

```diff
@@ -14,7 +14,7 @@ static const unsigned char REJECT_INVALID = 0x10;
 static const unsigned char REJECT_OBSOLETE = 0x11;
 static const unsigned char REJECT_DUPLICATE = 0x12;
 static const unsigned char REJECT_NONSTANDARD = 0x40;
-static const unsigned char REJECT_DUST = 0x41;
+// static const unsigned char REJECT_DUST = 0x41;
```

Comment:
Since this is part of BIP 61, I think it should remain as is even though it is unused.